radio_beam/beam.py

```diff
@@ -429,6 +435,16 @@ def as_kernel(self, pixscale):
                                           self.minor.to(u.deg).value/pixscale,
                                           self.pa.to(u.radian).value)
 
+    def as_struct_element(self, pixscale):
```

Comment:
OK, go with it (`as_tophat_kernel`).  However, I think in the present formulation, the area is not the same as the Gaussian kernel, and it should be.  The volume of a 2D gaussian is `2 pi sigma^2`, or `2*np.pi*major*minor/2.35**2`, so the effective major/minor axes of the tophat kernel should be `r_maj = major/sqrt(4*log(2))` if I'm doing my math right.